app/main.py

```diff
@@ -1,26 +1,30 @@
 import asyncio
 import os
 
-from dotenv import find_dotenv, load_dotenv
+import yaml
 from hypercorn import Config
 from hypercorn.asyncio import serve
 from piccolo.apps.user.tables import BaseUser
 from piccolo.engine import PostgresEngine
 from piccolo.engine.sqlite import SQLiteEngine
 from piccolo.table import create_db_tables
 from piccolo.table_reflection import TableStorage
-from piccolo_admin import create_admin
+from piccolo_admin.endpoints import TableConfig, create_admin
 from piccolo_api.encryption.providers import XChaCha20Provider
 from piccolo_api.mfa.authenticator.provider import AuthenticatorProvider
 from piccolo_api.mfa.authenticator.tables import (
     AuthenticatorSecret as AuthenticatorSecret_,
 )
 from piccolo_api.session_auth.tables import SessionsBase
 
-DB = SQLiteEngine()
-
+with open("config.yaml") as stream:
+    try:
+        admin_config = yaml.safe_load(stream)
+        BASE_CONFIG = admin_config.get("tables")
+    except yaml.YAMLError as exc:
+        raise exc
```

Comment:
I know you've merged this in, but in a future PR you could define the schema as a Pydantic model, and then pass the loaded YAML into that.

Reply:
You are right. I hadn't thought about Pydantic at all. With the Pydantic model parsing yaml will be much cleaner than it is now with dict. I will try to do it tomorrow. Thanks.